composer.json

```diff
@@ -4,11 +4,11 @@
 	"type": "wordpress-plugin",
 	"license": "GPL-2.0-only",
 	"require": {
-		"php": ">=5.6"
+		"php": ">=7.3"
 	},
 	"config": {
 		"platform": {
-			"php": "5.6.20"
+			"php": "7.3.33"
```

Comment:
Do you know what is the precise default of PHP we use at GoDaddy @fjarrett-godaddy  ? We could use that version (7.4.something) for the platform.

Reply:
@olafleur-godaddy 7.4.27

Reply:
Thank you!